content/guides/use-case/traefik.md

```diff
@@ -0,0 +1,235 @@
+---
+description: Use Traefik to easily route traffic between multiple containers or non-containerized workloads
+keywords: traefik, container-supported development
+title: HTTP routing with Traefik
+linktitle: HTTP routing with Traefik
+toc_max: 2
+---
+
+## Introduction
+
+During local development, it’s quite common to need to run multiple HTTP services. You might have both an API and a frontend app, a WireMock service to mock data endpoints, or a database visualizer (such as phpMyAdmin or pgAdmin). In many development setups, these services are exposed on different ports, which then requires you to remember what’s on what port but can also introduce other problems (such as CORS).
+
+A reverse proxy can dramatically simplify this setup by being the single exposed service and then routing requests to the appropriate service based on the request URL (either by path or hostname). [Traefik](https://traefik.io/traefik/) is a modern, cloud-native reverse proxy and load balancer that makes developing and deploying multi-service applications easier. This guide will show you how to use Traefik with Docker to enhance your development environment.
+
+In this guide, you will learn how to:
+
+1. Start Traefik with Docker
+2. Configure routing rules to split traffic between two containers
+3. Use Traefik in a containerized development environment
+4. Use Traefik to send requests to non-containerized workloads
+
+## Prerequisites
+
+The following prerequisites are required to follow along with this how-to guide:
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [Node.js](https://nodejs.org/en/download/package-manager) and [yarn](https://yarnpkg.com/)
+- Basic of Docker
+
+## Using Traefik with Docker
+
+One of the unique features of Traefik is its ability to be configured in many ways. When using the Docker provider, Traefik gets its configuration from other running containers using [labels](https://docs.docker.com/config/labels-custom-metadata/). Traefik will watch engine events (for container starts and stops), extract the labels, and update its configuration.
+
+The two most common Traefik-monitored labels will be ([view the full list here](https://doc.traefik.io/traefik/routing/providers/docker/)):
```

Comment:
Not a blocker, just a small thing I noticed. the double ()s cause an extra space in the deploy preview 
![image](https://github.com/user-attachments/assets/d460779e-b2c4-4191-af58-bd8f4a4b226a)

Reply:
Huh! Any idea how to resolve that seeing I have no space in the text itself? Could always just rewrite that segment to take it out of the ()s too.

Maybe something like this?

```suggestion
While there are [many Traefik-monitored labels](https://doc.traefik.io/traefik/routing/providers/docker/), the two most common will be:
```

Reply:
I like that fix!